python/lsst/ip/isr/ampOffset.py

```diff
@@ -27,9 +27,13 @@
 from lsst.meas.algorithms import (SubtractBackgroundTask, SourceDetectionTask)
 
 
+__all__ = ['AmpOffsetConfig', 'AmpOffsetTask']
```

Comment:
Doesn't main already have this change (but with different quotes)?

Reply:
(Ah, I see that the main one is a few lines above this -- the `__all__` should always be before the first import)

Reply:
It looks like the additions happened in parallel on different tickets, and I didn't notice in the diff because they're separated.  I'll go through and clean up the extras throughout.